### ISSUE TITLE
`lib-exoplayer-ima` skip ad functionality no longer supported by IMA

### ISSUE THREAD

**Issue by strangesource** [androidx/media#913]
### Version

Media3 1.2.0

### More version details

not version dependent

### Devices that reproduce the issue

any

### Devices that do not reproduce the issue

none

### Reproducible in the demo app?

Yes

### Reproduction steps

Very coarse steps, happy to provide more detail if needed:
- Extend sample to use `lib-exoplayer-ima` extension without an `adViewGroup` in order to use `createAudioAdDisplayContainer` 
- schedule client side skippable ad, e.g. `https://pubads.g.doubleclick.net/gampad/ads?iu=/21775744923/external/single_preroll_skippable&sz=640x480&ciu_szs=300x250%2C728x90&gdfp_req=1&output=vast&unviewed_position_start=1&env=vp&impl=s&correlator=` 
- play the ad until it is in skippable state (5 seconds for the sample tag)
- call `skipAd`

### Expected result

The ad should be skipped. The documentation on `skipAd` states that this should only be used for audio only ads. 

https://github.com/androidx/media/blob/f6fe90f30ba022c7e04e14a3dc5c28c568a4d1e2/libraries/exoplayer_ima/src/main/java/androidx/media3/exoplayer/ima/ImaAdsLoader.java#L505-L510

In the past this also worked for non-audio only ads but given the documentation I guess it's intended that it does not work if a view is present.

The root cause for this behaviour change is a change on IMA side, see ([1](https://groups.google.com/g/ima-sdk/c/ky-Q_pUXrIA/m/F_OEtx6tEgAJ), [2](https://groups.google.com/g/ima-sdk/c/1jDSlDS_pBI)) for reference.

### Actual result

Ad does not skip

### Media

Not dependent on main content, as skippable ad, `https://pubads.g.doubleclick.net/gampad/ads?iu=/21775744923/external/single_preroll_skippable&sz=640x480&ciu_szs=300x250%2C728x90&gdfp_req=1&output=vast&unviewed_position_start=1&env=vp&impl=s&correlator=` can be used.

### Bug Report

- [ ] You will email the zip file produced by `adb bugreport` to android-media-github@google.com after filing this issue.

**Comment by marcbaechinger** [comment]
Thanks for reporting. 

The call you mention is delegated to `com.google.ads.interactivemedia.v3.api.AdsManager.skip()` of the SDK which then is expected to send us ad events that skip the ad. 
If the SDK doesn't allow this anymore, we can't do much I'm afraid. 

I think the references you link to the IMA issue tracker is the right place to discuss this. We are IMA SDK users just as you are. The IMA team needs to sort this out, we can not really help with this I'm afraid.

**Comment by marcbaechinger** [comment]
Sorry, can you clarify whether you report this is not working for audio ads either? Or is this only about skipping not working for video ads anymore?

**Comment by strangesource** [comment]
Sorry for not being clear, the intention of this issue is to align the API with what IMA allows, not to make it work again. 

It also doesn't work for audio only ads - at least in my tests.

**Comment by marcbaechinger** [comment]
Agreed we should not promise something with our API that's not possible. We will check this internally and then adjust API or documentation if needed.

Thanks for clarifying.
